lib/teiserver/bridge/discord_bridge_bot.ex

```diff
@@ -127,13 +127,13 @@ defmodule Teiserver.Bridge.DiscordBridgeBot do
 
     response = CommandLib.handle_command(interaction, options_map)
 
-    # response = case data.name do
-    #   "textcb" ->
-    #     Teiserver.Bridge.TextcbCommand.execute(interaction, options_map)
+    response = case data.name do
+      "textcb" ->
+        Teiserver.Bridge.TextcbCommand.execute(interaction, options_map)
 
-    #   _ ->
-    #     nil
-    # end
+      _ ->
+        nil
+    end
```

Comment:
why the uncomment here? It seems this is going to override the response.
I'm not sure what is this `textcb` command, but at least it should be

```elixir
    response = case data.name do
      "textcb" ->
        Teiserver.Bridge.TextcbCommand.execute(interaction, options_map)
      _ ->
        CommandLib.handle_command(interaction, options_map)
    end

```

Reply:
Text callbacks are the  discord commands tha start with / and have autocomplete, for teiserver they return some premade responses on commonly asked questions. This isn't working for some reason so I was trying out different things.